### ISSUE TITLE
Notes field on LentItem

### ISSUE THREAD

**Issue by tribut** [opacapp/opacclient#654]
While implementing a simple cli app using libopac, I noticed there is no way to access the additional information displayed in the account page ("Dieses Medium wurde bereits 3 mal verlängert", "Eine Verlängerung ist noch nicht möglich"):

![Screenshot_20230805_094516_Firefox](https://github.com/opacapp/opacclient/assets/719105/f867592a-8ad6-4947-bb3b-46f944490ab5)

My proposal would be to add an additional string field to LentItem (notes? statusText?). I could provide a PR that implements this for SISIS (Stb. Chemnitz), but wanted to make sure this is something you are interested in.

Thanks for all your work!

**Comment by raphaelm** [comment]
Yes, I think we could add something like that. My gut says ``statusNote``, but I don't have strong feelings about this.
